picky/Cargo.toml

```diff
@@ -68,23 +68,33 @@ aes-gcm = { version = "0.10", optional = true }
 aes = { version = "0.8.2", optional = true }
 aes-kw = { version = "0.2.1", optional = true, features = ["std"] }
 ctr = { version = "0.9.2", optional = true }
-cbc = { version = "0.1.2", optional = true }
-bcrypt-pbkdf = { version = "0.9.0", optional = true }
+cbc = { version = "0.1.2", optional = true, features = ["alloc"] }
+bcrypt-pbkdf = { version = "0.10.0", optional = true }
+des = { version = "0.8", optional = true }
+rc2 = { version = "0.8", optional = true }
+pbkdf2 = { version = "0.12", optional = true }
+hmac = { version = "0.12", optional = true }
+
+ucs2 = "0.3"
 
 [dev-dependencies]
 pretty_assertions = "1.3.0"
 hex = "0.4.3"
 cfg-if = "1.0.0"
 ring = "0.16.20"
 rstest = "0.12.0"
+expect-test = "1"
+tempfile = "=3.3"
```

Comment:
Nit: a comment explaining why we are pinning this dependency could help later maintenance

Reply:
Yep, agreed, I'll add that

Reply:
Done